### DISCUSSION TITLE
识别错误该如何处理？

### DISCUSSION THREAD

**Discussion by jimmymonopole** [NateScarlet/auto-derby#240] [Q&A]
[auto_derby.log](https://github.com/NateScarlet/auto-derby/files/8101503/auto_derby.log)
![屏幕截图 2022-02-19 134035](https://user-images.githubusercontent.com/47505247/154788038-b4bb2229-05c7-4fed-b617-7917259069e6.jpg)
不太了解opencv，他一直把这个“后半”识别成“n半”这怎么可能的？我现在把context.py里那个字典加了个“n半”勉强能用了，但不知道为啥会有这种问题。

**Comment by NateScarlet** [comment]
这个应该是问 “后” 识别对不对的时候你 回答了n 然后问正确的是什么你又回答了 n

看 #102

把你本地数据 data/ocr_labels.csv 中对应 n 的记录删掉应该就行  游戏内好像没有带n的文本需要识别
